### DISCUSSION TITLE
Is there a way to periodically update the SSR bundle?

### DISCUSSION THREAD

**Discussion by flppv** [vercel/next.js#14790] [Help] [answered]
I have an app, it utilizes SSR approach for some pages (e.g. fetch reviews from DB). 

My problem: until I rebuild the bundle (with Circle CI) it doesn't update reviews page with fresh reviews, that can be added a few times per day. 

Is there a way to rebuild the bundle once per day or I am doing something incorrectly?

That's how I fetch them:

```
export async function getStaticProps() {
  const res = await axios.get(`${process.env.apiUrl}/public/review`)
  const reviews = res.data

  return {
    props: {
      reviews,
    },
  }
}

export default ReviewsPage
```

**Comment by timneutkens** [comment, marked as answer]
You'll likely want to leverage https://github.com/vercel/next.js/discussions/11552
